src/main/java/com/fortify/plugin/jenkins/steps/FortifyCloudScanStep.java

```diff
@@ -28,13 +27,13 @@ public String getResolvedScanArgs(TaskListener listener) {
     protected String getCloudScanExecutable(Run<?, ?> build, FilePath workspace, Launcher launcher,
                                          TaskListener listener) throws InterruptedException, IOException {
         listener.getLogger().println("Checking for cloudscan executable");
-        return getExecutable("cloudscan" + (launcher.isUnix() ? "" : ".bat"), true, build, workspace, launcher,
-                listener, null);
+        return getExecutable("cloudscan" + (launcher.isUnix() ? "" : ".bat"), build, workspace,
```

Comment:
Thanks for your input. I don't believe there are any restrictions at the moment for the version of Fortify_SCA_and_Apps because it is assigned manually through setting FORTIFY_HOME env variable and currently there is no logic under the hood checking that version. Although, I think it's not safe to drop that possibility off completely, as some clients may still use it and they might be surprised when they find themselves being not allowed to do that anymore. For now, I would prefer to add a check if target Fortify_SCA_and_Apps has cloud scan tool and if not, then throw an exception saying that instead dropping cloud scan tool off in order to avoid rollback requests as in case with field for FPR in ‘Local Translation & Local Scan’ Fortify Assessment option. I have added that change.